<ui version="4.0" >
 <class>dEditColumns</class>
 <widget class="Q3MainWindow" name="dEditColumns" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Columns</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="3" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Name:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eName" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" >
     <layout class="QVBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>44</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bAddAction" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bRemoveAction" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bMoveUp" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bMoveDown" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>21</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="2" >
     <widget class="Q3Frame" name="frmFields" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="Q3GroupBox" name="gbFields" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Columns</string>
      </property>
      <property name="midLineWidth" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="Action" >
   <property name="iconText" >
    <string>Action</string>
   </property>
   <property name="name" stdset="0" >
    <string>Action</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >atreeitems.h</include>
  <include location="local" >aliaseditor.h</include>
  <include location="local" >mdtree.h</include>
  <include location="local" >mainform.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>bAddAction</sender>
   <signal>clicked()</signal>
   <receiver>dEditColumns</receiver>
   <slot>bAddAction_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bRemoveAction</sender>
   <signal>clicked()</signal>
   <receiver>dEditColumns</receiver>
   <slot>bRemoveAction_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bMoveUp</sender>
   <signal>clicked()</signal>
   <receiver>dEditColumns</receiver>
   <slot>bMoveUp_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bMoveDown</sender>
   <signal>clicked()</signal>
   <receiver>dEditColumns</receiver>
   <slot>bMoveDown_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
